src/iiiftilesource.js

```diff
@@ -181,20 +182,34 @@ $.extend( $.IIIFTileSource.prototype, $.TileSource.prototype, /** @lends OpenSea
     },
 
     /**
-     * Return the tileSize for the given level.
+     * Return the tileWidth for the given level.
      * @function
      * @param {Number} level
-    */
+     */
+    getTileWidth: function( level ) {
```

Comment:
Just tested with the image @azaroth42 kindly provided (http://dlss-dev-azaroth.stanford.edu/services/iiif_ns/f1rc/info.json). Screenshot of result follows, with debugMode=true to display the non-square tiles. :smile:  Looks correct to me?

![screen shot 2015-07-14 at 10 44 02 am](https://cloud.githubusercontent.com/assets/597044/8676101/57f32244-2a15-11e5-90dc-cd34244416f7.PNG)